images/macos/software-report/SoftwareReport.Browsers.psm1

```diff
@@ -1,3 +1,5 @@
+Import-Module "$PSScriptRoot/../helpers/SoftwareReport.Helpers.psm1" -DisableNameChecking
```

Comment:
I don't think that we need this import here. `SoftwareReport.Helpers.psm1` in software generator file